docs/book/src/getting-started.md

```diff
@@ -1,3 +1,645 @@
+let's check the status using `kubectl get nodes`:
+
+```bash
+kubectl --kubeconfig=./capi-quickstart.kubeconfig get nodes
+```
+
+```bash
+NAME                                          STATUS   ROLES           AGE   VERSION
+capi-quickstart-g2trk-9xrjv                   Ready    control-plane   12m   v1.26.0
+capi-quickstart-g2trk-bmm9v                   Ready    control-plane   11m   v1.26.0
+capi-quickstart-g2trk-hvs9q                   Ready    control-plane   13m   v1.26.0
+capi-quickstart-md-0-55x6t-5649968bd7-8tq9v   Ready    <none>          12m   v1.26.0
+capi-quickstart-md-0-55x6t-5649968bd7-glnjd   Ready    <none>          12m   v1.26.0
+capi-quickstart-md-0-55x6t-5649968bd7-sfzp6   Ready    <none>          12m   v1.26.0
+```
+
+### Clean Up
+
+Delete workload cluster.
+```bash
+kubectl delete cluster capi-quickstart
+```
+<aside class="note warning">
+
+IMPORTANT: In order to ensure a proper cleanup of your infrastructure you must always delete the cluster object. Deleting the entire cluster template with `kubectl delete -f capi-quickstart.yaml` might lead to pending resources to be cleaned up manually.
+</aside>
+
+Delete management cluster
+```bash
+kind delete cluster
+```
+
+## Next steps
+
+See the [clusterctl] documentation for more detail about clusterctl supported actions.
+
+<!-- links -->
+[Experimental Features]: https://cluster-api.sigs.k8s.io/tasks/experimental-features/experimental-features.html
+[AWS provider prerequisites]: https://cluster-api-aws.sigs.k8s.io/topics/using-clusterawsadm-to-fulfill-prerequisites.html
+[AWS provider releases]: https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases
+[Azure Provider Prerequisites]: https://capz.sigs.k8s.io/topics/getting-started.html#prerequisites
+[bootstrap cluster]: ../reference/glossary.md#bootstrap-cluster
+[capa]: https://cluster-api-aws.sigs.k8s.io
+[capv-upload-images]: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/master/docs/getting_started.md#uploading-the-machine-images
+[clusterawsadm]: https://cluster-api-aws.sigs.k8s.io/clusterawsadm/clusterawsadm.html
+[clusterctl generate cluster]: https://cluster-api.sigs.k8s.io/clusterctl/commands/generate-cluster.html
+[clusterctl get kubeconfig]: ../clusterctl/commands/get-kubeconfig.md
+[clusterctl]: https://cluster-api.sigs.k8s.io/clusterctl/overview.html
+[Docker]: https://www.docker.com/
+[GCP provider]: https://github.com/kubernetes-sigs/cluster-api-provider-gcp
+[Hetzner provider]: https://github.com/syself/cluster-api-provider-hetzner
+[IBM Cloud provider]: https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud
+[infrastructure provider]: ../reference/glossary.md#infrastructure-provider
+[kind]: https://kind.sigs.k8s.io/
+[KubeadmControlPlane]: ../developer/architecture/controllers/control-plane.md
+[kubectl]: https://kubernetes.io/docs/tasks/tools/install-kubectl/
+[management cluster]: ../topics/reference/glossary.md#management-cluster
```

Comment:
anything with topics/reference/ folders won't work here as those were pointing to local file system in CAPI repo.

Reply:
Please check the failing md-lint checker job, it would show the same error

Reply:
> anything with topics/reference/ folders won't work here as those were pointing to local file system in CAPI repo.

Thanks @Ankitasw for the suggestions
Actually we have a glossary.md in our CAPA repo and inside that we are using CAPI embedded glossary.md. And it is also working fine on my localhost also.

Reply:
Then I think any embed is not working, and we have to make our own glossary.

Reply:
Can you please check it by building book on your localhost? 
And then please list out all the required changes.

Reply:
https://drive.google.com/file/d/1DEohfdYx6iZH2wJDRUxxn08xW-VFwwJb/view?usp=share_link

Reply:
@Ankitasw check in it. I think embedded glosarry.md is working fine.

Reply:
Or I am not getting your point. Can you please explain that?

Reply:
ok never mind!! In preview it was looking differently.